src/qml/CMakeLists.txt

```diff
@@ -1,16 +1,17 @@
 find_package(${QT_PKG}QuickCompiler QUIET)
 find_package(${QT_PKG} COMPONENTS Quick)
 
+file(GLOB_RECURSE QML_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.qml")
```

Comment:
Can we merge https://github.com/opengisch/QField/pull/3106 and rebase?

Reply:
@m-kuhn , sounds good, approved that PR just now.